Description: Add unifont-viewer.1 to list of man pages to install.
The man page unifont-viewer.1 was omitted from the MAN1PAGES variable
in man/Makefile, and so was not installed.  This patch corrects that.
--- a/man/Makefile
+++ b/man/Makefile
@@ -4,9 +4,9 @@ SHELL = /bin/sh
 MAN1PAGES = bdfimplode.1 hex2bdf.1 hex2sfd.1 \
 	hexbraille.1 hexdraw.1 hexmerge.1 hexkinya.1 johab2ucs2.1 \
 	unibdf2hex.1 unibmp2hex.1 unicoverage.1 unidup.1 \
-	unifontpic.1 unifontchojung.1 unifontksx.1 unigencircles.1 \
-	unigenwidth.1 unihex2bmp.1 unihex2png.1 unihexgen.1 unihexfill.1 \
-	unipagecount.1 unipng2hex.1
+	unifontpic.1 unifontchojung.1 unifontksx.1 unifont-viewer.1 \
+	unigencircles.1 unigenwidth.1 unihex2bmp.1 unihex2png.1 unihexgen.1 \
+	unihexfill.1 unipagecount.1 unipng2hex.1
 
 MAN5PAGES = unifont.5
 
